Code/GraphMol/ChemTransforms/MolFragmenter.cpp

```diff
@@ -418,7 +418,8 @@ std::vector<std::pair<Bond *, std::vector<int>>> getNbrBondStereo(
     RWMol &mol, const Bond *bnd) {
   PRECONDITION(bnd, "null bond");
   // loop over neighboring double bonds and remove their stereo atom
-  std::vector<std::pair<Bond *, std::vector<int>>> res;
+  std::vector<std::pair<Bond *, std::vector<int
+    res;
```

Comment:
It looks like something went badly with one of the merges:
```suggestion
  std::vector<std::pair<Bond *, std::vector<int>>>  res;
```